### PR TITLE
[Feature] Updates to package to publish on npmjs

### PR DESCRIPTION
**Pull request by SferaDev**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2690,6 +2701,20 @@ Make the sheet the active sheet in the workkbok.
 | --- | --- | --- |
 | active | <code>boolean</code> | Must be set to `true`. Deactivating directly is not supported. To deactivate, you should activate a different sheet instead. |
 
+<a name="Sheet+sharedFormulas"></a>
+
+#### sheet.sharedFormulas() ⇒ <code>boolean</code>
+Gets a value indicating whether the sheet is the active sheet in the workbook.
```
Comment (tokland) [inline README.md:2707]: This seems c&p of sheet.active

File: scripts/publish.sh [hunk 2]
```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e -u
+
+yarn build
```
Comment (tokland) [inline scripts/publish.sh:4]: [very minor] Maybe move this "yarn build" to the package.json, so the script only publishes.

**Review by tokland — COMMENTED** [review]
Looks good! I just found a text to fix in the README.

File: scripts/publish.sh (hunk 2 above)
Comment (SferaDev) [reply scripts/publish.sh]: Done

File: README.md (hunk 1 above)
Comment (SferaDev) [reply README.md]: Fixed

**Pull request merged**